csharp_dotnetcore/13.Basic-Bot-Template/BotServices.cs

```diff
@@ -38,7 +38,7 @@ public BotServices(BotConfiguration botConfiguration)
 
```

Comment:
Remove all files under \botbuilder-samples\csharp_dotnetcore and add these under \botbuilder-samples\samples\csharp_dotnetcore.